docs/eBpfStringsAPI.md

```diff
@@ -0,0 +1,91 @@
+# eBPF Strings Library API proposal
+
+## Overview
+
+A majority of use cases and examples of eBPF functions are focused on operating on fixed fields in buffers; network structures with known sizes and offsets, bit-fields, and the like. The programs we can write with the available tooling are powerful, but fall short of the needs of Application-layer customers like HTTP servers, where string operations are mandatory to interact with HTTP requests and responses.
```

Comment:
Can you split this (and other long lines) into multiple source lines? 
https://github.com/microsoft/ebpf-for-windows/blob/main/docs/DevelopmentGuide.md#formatting-notes says:
"Source lines MUST NOT exceed 120 columns."
(This isn't just for source code, but is so that github diffs work well for all files.)